### DISCUSSION TITLE
installazione elementaryos stable su portatile con windows 10

### DISCUSSION THREAD

**Discussion by gianfoga** [elementary/triage#627] [Q&A]
buon giorno, scusate se sono poco pratico di so , sono un utente windows  datato 1941 !!!!!????;
-ho salvato i dati esistenti sul mio pc e creato una partizione del disco dove vorrei installare elementary;
-ho scaricato il so in oggetto, che usavo molti anni fa, e trasferito su una chiavetta avviabile;
-avviata la chiavetta per installare il so mi dice che cancellerà tutto e installerà il so.
Chiedo conferma che la formattazione riguarderà la sola partizione creata appositamente per installare il nuovo so in dual bot con Windows.
Ringrazio per l'aiuto che riceverò
Giancarlo

**Comment by teamcons** [comment]
Buon giorno

È necessario scegliere l'opzione - partizionamento personalizzato
ti permetterà di scegliere quale partizione formattare e su cui installare il sistema operativo elementare.

solo la partizione per la quale hai selezionato “formattazione” avrà i suoi dati cancellati.
l'avvertenza sulla cancellazione dei dati vale per le partizioni che si è scelto di utilizzare e formattare.

non cancellerà le partizioni che avete lasciato da sole.

assicuratevi comunque di aver salvato i vostri file personali altrove rispetto al computer

mi scuso per una risposta scritta con DeepL

**Comment by gianfoga** [comment]
grazie proverò a fare come indicato anche se non sono esperto di
partizionamento linux
saluti

Il giorno sab 23 nov 2024 alle ore 20:55 Stella and Charlie <
***@***.***> ha scritto:

**Reply by teamcons** [reply]
scegliere semplicemente la partizione facendo clic su di essa
indicare che si desidera utilizzarla per “/” o “root”.
formattarla (scegliere “ext4” se non è quella predefinita)

########

Nei computer moderni si raccomanda spesso di fare spazio a una partizione da 512 MB e di usarla per “/boot/efi”.

è uno spazio riservato a “grub”, il software che permette di scegliere tra windows e linux all'avvio, per mantenere il registro di sistema che contiene tutto ciò che serve per entrambi i sistemi. questo è chiamato bootloader

Avere i due sistemi fianco a fianco è un po' complicato: gli aggiornamenti di Windows a volte sostituiscono grub all'avvio con il grub di Windows (che vuole avviare solo Windows). Una partizione separata aiuta a risolvere questo tipo di problema.

**Comment by gianfoga** [comment]
grazie ,proverò

Il giorno dom 24 nov 2024 alle ore 11:26 Stella and Charlie <
***@***.***> ha scritto:
